.vscode/settings.json

```diff
@@ -36,5 +36,6 @@
     "editor.formatOnSave": true,
     "editor.formatOnPaste": true,
     "editor.formatOnSaveMode": "file"
-  }
+  },
+  "cSpell.words": ["redelegate"]
```

Comment:
can remove as we're not maintaining a list here. I think I added via the IDE it out of habit

Reply:
fine leaving, if you see more feel free to add.